lib/gateways/internal-api.ts

```diff
@@ -43,4 +46,42 @@ export const createApplication = async (applicants: Resident[]): Promise<any> =>
   }
 }
 
-// TODO: update application
+/**
+ * Update application
+ * @param {Resident[]} applicants The applicant information
+ * @returns {any} The updated application
+ */
+export const updateApplication = async (applicants: Resident[]): Promise<any> => {
+  try {
+    const personal = applicants[0].formData["personal-details"]
```

Comment:
This code could be extracted out as it will likely be the same for create/update.

Reply:
Not entirely sure I understand what you mean.

Reply:
Currently this code is repeated between create/update methods, we could move to a helper or similar.

Reply:
Good shout. Created a helper function now